src/Components/Footer/index.js

```diff
@@ -1,90 +1,55 @@
+                       <FooterLink to="/develop">Destinations</FooterLink>
+                       <FooterLink to="/develop">Sponsorships</FooterLink>
+                </FooterLinkItems>
+            </FooterLinksWrapper>
+            <FooterLinksWrapper>
+                <FooterLinkItems className='videos'>
+                    <FooterLinkTitle>Videos</FooterLinkTitle>
+                    <FooterLinkTitleHr className='hr' />
+                       <FooterLink to="/develop">Submit Video</FooterLink>
+                       <FooterLink to="/develop">Ambassdors</FooterLink>
+                       <FooterLink to="/develop">Agency</FooterLink>
+                       <FooterLink to="/develop">Influencer</FooterLink>
+                </FooterLinkItems>
           </FooterLinksWrapper>
         </FooterLinksContainer>
         <SocialMedia>
           <SocialMediaWrap>
-          <FooterLinkItems className='socials'>
-  <FooterLink className="icons youtube" aria-label="Visit us on Youtube" title="Youtube (External Link)" style={{ backgroundColor: "#FF0000" }}>
-    <FaYoutube style={{ fontSize: '22px' }} />
-  </FooterLink>
-  <FooterLink className="icons twitter" aria-label="Visit us on Twitter" title="Twitter (External Link)" style={{ backgroundColor: "#1DA1F2" }}>
-    <FaTwitter style={{ fontSize: '22px' }} />
-  </FooterLink>
-  <FooterLink className="icons linkdn" aria-label="Visit us on Linkedin" title="Linkedin (External Link)" style={{ backgroundColor: "#0077B5" }}>
-    <FaLinkedin style={{ fontSize: '22px' }} />
-  </FooterLink>
-  <FooterLink className="icons insta" aria-label="Visit us on Instagram" title="Instagram (External Link)" style={{ backgroundColor: "#E4405F" }}>
-    <FaInstagram style={{ fontSize: '22px' }} />
-  </FooterLink>
-  <FooterLink className="icons fb" aria-label="Visit us on Facebook" title="Facebook (External Link)" style={{ backgroundColor: "#1877F2" }}>
-    <FaFacebook style={{ fontSize: '22px' }} />
-  </FooterLink>
-  <a href="https://github.com/MonalikaPatnaik/UMatter" target="_blank" rel="noopener noreferrer" className="icons github" aria-label="Visit us on Github" title="Github (External Link)" style={{ backgroundColor: "black", display: "inline-block", verticalAlign: "middle", marginTop: "-7px", transition: "color 0.3s ease, background-color 0.3s ease, transform 0.3s ease" }}>
-    <FaGithub className="github-icon" style={{ fontSize: '34px', color: "white", maskImage: "url(#github-mask)" }} />
-    <svg style={{ display: "none" }}>
-      <mask id="github-mask">
-        <rect width="100%" height="100%" fill="white" />
-        <FaGithub className="github-icon" style={{ fontSize: '34px', color: "black" }} />
-      </mask>
-    </svg>
-  </a>
-  <style>{`
-    .github-icon:hover {
-      transform: translateY(-8px);
-    }
-  `}</style>
-</FooterLinkItems>
-            <FooterLinkItems className='socials'>
-              {/* Social media icons */}
```

Comment:
Also, don't change the footer.

Reply:
some conflicts were there and while resolving and merging them i guess i made misake

Reply:
i'll do one thing i'll close this pr and make another one after taking pull again and work with update changes
sorry for thr inconvience